app/crowd/models.py

```diff
@@ -0,0 +1,66 @@
+import uuid
+from typing import Any, Optional
+
+from django.contrib.auth.models import AbstractUser, UserManager
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class CrowdUserManager(UserManager):  # type: ignore
+    """Manager allowing username OR first_name as required argument."""
+
+    def create_user(
+        self,
+        username: str = "",
+        email: Optional[str] = None,
+        password: Optional[str] = None,
+        **extra_fields: Any,
+    ) -> AbstractUser:
+        """Create user requiring username or first_name."""
+        return super().create_user(  # type: ignore
+            username, email, password, **extra_fields
+        )
+
+    def _create_user(
```

Comment:
Why do we need two methods? Private and a public? Probably I missed something...

Reply:
These are methods from original `UserManager` class.